components/StateHandlers/HydrationStateHandler.tsx

```diff
@@ -33,10 +33,10 @@ export default function HydrationStateHandler() {
           );
 
           const results = await Promise.allSettled([
-            getXmtpClient(account),
+            // This will handle creating the client and setting the conversation list from persistence
             fetchPersistedConversationListQuery({ account }),
-            prefetchInboxIdQuery({ account }),
           ]);
+          prefetchInboxIdQuery({ account });
```

Comment:
Moved this out as it's a prefetch, we don't want to await it

Reply:
It should be cached better now, so I think it will be a better experience, where previously it was not cached